ResumableUpload.VB.Sample/README.html

```diff
@@ -0,0 +1,42 @@
+<html>  
+<title>Google .NET Client API &ndash; ResumableUpload.VB.Sample</title>  
+<body>  
+<h2>Instructions for the Google .NET Client API &ndash; ResumableUpload.VB.Sample</h2>  
+  
+<h3>Browse Online</h3>  
+  
+<ul>  
+  <li><a href="http://code.google.com/p/google-api-dotnet-client/source/browse/?repo=samples#hg%2FResumableUpload.VB.Sample">Browse Source</a>,  
+   or main file <a href="http://code.google.com/p/google-api-dotnet-client/source/browse/ResumableUpload.VB.Sample/ResumableUpload.vb?repo=samples">ResumableUpload.vb</a></li>  
+</ul>  
+   
+<h3>1. Checkout Instructions</h3>  
+ 
+<p><b>Prerequisites:</b> Install Visual Studio, and <a href="http://mercurial.selenic.com/">Mercurial</a>.</p>  
+  
+<pre><code>cd <i>[someDirectory]</i>  
+hg clone https://code.google.com/p/google-api-dotnet-client.samples/ google-api-dotnet-client-samples</code></pre>  
+  
+<h3>2. API access Instructions</h3>  
+<p><b>Important:</b> after checking out the project, compiling and running it, you need to do the following: 
+<ul>  
+<li>Visit the <a href="https://code.google.com/apis/console/">Google APIs console</a></li> 
+	<li>If this is your first time, click "Create project..."</li> 
+	<li>Otherwise, click on the drop down under the "Google APIs" logo at 
+	    the top left, and click "Create..." under "Other projects"</li> 
+	<li>Click on "API Access", and then on "Create an OAuth 2.0 Client ID...".</li> 
+	<li>Enter a product name and click "Next".</li> 
+	<li>Select "Installed application" and click "Create client ID".</li> 
+	<li>In the newly created "Client ID for installed applications", copy the client ID and client secrets into the ResumableUpload.vb file.</li> 
+	<li><a href="https://code.google.com/apis/console/?api=drive">Activate</a> the YouTube Data API v3 for your project.</li>  
+</ul> 
+</p> 
+  
+<h3>Setup Project in Visual Studio</h3>  
```

Comment:
Same comment as the previous README. Things like "Execute the .exe in the bin\Debug" instructions are probably unnecessary.